modulemd/include/private/modulemd-yaml.h

```diff
@@ -801,6 +1243,22 @@ skip_unknown_yaml (yaml_parser_t *parser, GError **error);
     }                                                                         \
   while (0)
 
+/**
+ * EMIT_HASHTABLE_KEY_VALUES_IF_NON_EMPTY:
+ * @emitter: (inout): A libyaml emitter object that is positioned at the start
+ * of where a new mapping will be written.
+ * @error: (out): A #GError that will return the reason for any error.
+ * @key: (in): The name to be used as the identifier for the output mapping.
+ * @table: (in): The #GHashTable that is to be output.
```

Comment:
Indicate that this function only works on a `GHashTable` where both the keys and values are strings.